ftw/workspace/profiles/contact/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<metadata>
+  <dependencies>
```

Comment:
You should add a `<version>` here, or an declare an upgrade step directory. Otherwise this profile will not be upgradeable because we do not know whether it was installed.